testing/config_classification.yaml

```
@@ -22,7 +22,7 @@ model:
   final_layer: softmax
 nested_training:
   testing: -2
-  validation: -2
+  validation: 3
```

```suggestion
  validation: -3
```

Single round of validation, otherwise the tests take too long.

---

I see, but  I think the coverage is decreased to lack of n-folds. Would keeping it as 2 be okay?

---

Add both cases in the test.